frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/user-templates.tsx

```diff
@@ -16,7 +16,14 @@ import { iGetName } from '../../selectors/immutable/selectors';
 import { iGetFieldValue } from '../../selectors/immutable/vm-settings';
 
 export const UserTemplates: React.FC<UserTemplatesProps> = React.memo(
-  ({ userTemplateField, userTemplates, commonTemplates, onChange, openshiftFlag }) => {
+  ({
+    userTemplateField,
+    userTemplateName,
```

Comment:
it is very confusing to read a variable like this in this context.
Can we rename it here?

like: ` forceSingleUserTemplateName` ?